### PR TITLE
[DO NOT MERGE, YET] Changes made for realising coverage frontier visualisations

### PR DESCRIPTION
**Pull request by BharathMonash**
Collecting segment and function specific information from the coverage_summary.json (for each benchmark-fuzzer pair for each trial) for visualising coverage frontier plots . The segment and function specific data is being collected in a DataFrame and being dumped on drive as a .csv for reporting and plotting.

Well, other than that, recording timestamp information for the segments or function being covered could be used to make interactive plots showing the coverage frontier over time. Any Suggestion on how to include the timestamp information to the coverage summary being generated such that information for every 15 mins is included?

### CODE REVIEW

**Comment by BharathMonash** [comment]
I have signed the CLA.

**Comment by BharathMonash** [comment]
@googlebot, I consent.

**Comment by BharathMonash** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Ffuzzbench%2Fpull%2F778) for more info**.

Hi @mboehme, consent is needed for your commits.

**Comment by mboehme** [comment]
@googlebot I consent.

File: experiment/coverage_utils.py [hunk 1]
```diff
@@ -165,9 +196,39 @@ def generate_coverage_report(self):
...
+                    trial_id=trial_id)
+            segment_df = pd.concat([segment_df, seg_df], axis=0,
+                                   ignore_index=True)
+            function_df = pd.concat([function_df, func_df], axis=0,
+                                    ignore_index=True)
+
+        segment_csv_src = os.path.join(self.benchmark_fuzzer_measurement_dir,
+                                       'segments.csv')
+        function_csv_src = os.path.join(self.benchmark_fuzzer_measurement_dir,
+                                        'functions.csv')
```
Comment (mboehme) [inline experiment/coverage_utils.py:228]: This PR is about generating two CSV files containing coverage information on segments and functions.
* The format of segment.csv is `benchmark, fuzzer, trial_id, filename, line, col, hits`
* The format of functions.csv is `benchmark, fuzzer, trial_id, function_name, hits`

We currently have one file for *each* (benchmark, fuzzer) in `benchmark_fuzzer_dir`, but eventually we would like to have one file for *all* benchmarks and fuzzers in the parent directory of `benchmark_fuzzer_dir`. 

We currently don't have time stamps. Any suggestions to add those are welcome.

Comment (mboehme) [reply experiment/coverage_utils.py:228]: If file size is a concern, we could move names (benchmark, fuzzer, file_name, function_name) into a separate file and reference them by ID.

**Comment by inferno-chromium** [comment]
Not ready for review right ? WIP ?

**Comment by BharathMonash** [comment]
> Not ready for review right ? WIP ?

Yup, thats right, work in progress.

**Pull request closed without merging**